### DISCUSSION TITLE
Cannot find dotnet/0.2.2 on Nuget.org

### DISCUSSION THREAD

**Discussion by mirceacistian-gep** [microsoft/autogen#4539] [Q&A] [answered]
Hi all,

I was wondering if there will be a release of the 0.2.2 package for dotnet on https://www.nuget.org/packages/AutoGen?

Seeing that Semantic Kernel packages were updated I think it will help with some issues we're having with dependencies on System.ClientModel when using a newer version of SK with Autogen.

https://microsoft.github.io/autogen-for-net/release_note/0.2.2.html

Thanks!

**Comment by rysweet** [comment]
@LittleLittleCloud any updates on this?
@mirceacistian-gep I think for now you are best served by grabbing the source. 
the 0.4 version works with newer versions of SK and also with Microsoft.Extensions.AI

**Comment by LittleLittleCloud** [comment, marked as answer]
@mirceacistian-gep 
Currently we are working on real-signing pakcages so haven't push the latest package to nuget.org yet., you can consume it in our nightly build though. It depends on sk 1.29.0 so might be helpful with the dependency issues
- https://dev.azure.com/AGPublish/AGPublic/_artifacts/feed/AutoGen-Nightly

**Comment by mirceacistian-gep** [comment]
Thank you @rysweet  @LittleLittleCloud
I'll check out both options.
